modules/gsp-cluster/harbor.tf

```diff
@@ -1,172 +0,0 @@
-resource "aws_iam_role" "harbor" {
-  name               = "${var.cluster_name}-harbor"
-  description        = "Role the harbor process assumes"
-  assume_role_policy = data.aws_iam_policy_document.trust_kiam_server.json
-}
-
-data "aws_iam_policy_document" "harbor-s3" {
-  statement {
-    actions = [
-      "s3:*",
-    ]
-
-    resources = [
-      aws_s3_bucket.ci-system-harbor-registry-storage.arn,
-      "${aws_s3_bucket.ci-system-harbor-registry-storage.arn}/*",
-    ]
-  }
-}
-
-resource "aws_iam_policy" "harbor-s3" {
-  name        = "${var.cluster_name}-harbor-s3"
-  description = "Policy for the harbor s3 access"
-  policy      = data.aws_iam_policy_document.harbor-s3.json
-}
-
-resource "aws_iam_policy_attachment" "harbor-s3" {
-  name       = "${var.cluster_name}-harbor-s3"
-  roles      = [aws_iam_role.harbor.name]
-  policy_arn = aws_iam_policy.harbor-s3.arn
-}
-
-resource "random_password" "concourse_password" {
```

Comment:
:thinking: I think concourse might still want this

Reply:
Yes... What on earth is it doing in harbor.tf? Will shift to another gsp-cluster file.